controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -282,15 +303,7 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		// Watcher for storagecluster required to update
 		// ocs-operator-config configmap if storagecluster spec changes
 		Watches(
-			&ocsv1.StorageCluster{},
-			handler.EnqueueRequestsFromMapFunc(
-				func(context context.Context, obj client.Object) []reconcile.Request {
-					return []reconcile.Request{{
-						NamespacedName: InitNamespacedName(),
-					}}
-				},
-			),
-			builder.WithPredicates(predicate.GenerationChangedPredicate{}),
+			&ocsv1.StorageCluster{}, enqueueOCSInit, builder.WithPredicates(predicate.GenerationChangedPredicate{}),
```

Comment:
If this is a multi-line statement then each item should be on its own raw, if not everything should be on the `watches` raw (but I think the line is going to be too long)

```suggestion
			&ocsv1.StorageCluster{}, 
			enqueueOCSInit, 
			builder.WithPredicates(predicate.GenerationChangedPredicate{}),
```